.bazelrc

```diff
@@ -0,0 +1 @@
+common --experimental_enable_bzlmod
```

Comment:
If you are using Bazel 6.0 or later, please just use `--enable_bzlmod` since it's no longer experimental.

Reply:
there should be a `.bazelversion` file added to this repo as well, or you'll never be able to reason about "if you are using Bazel 6.0"